src/components/MenuPanel/Actions.js

```diff
@@ -0,0 +1,68 @@
+// @flow
+
+import React from 'react'
+import { t } from 'ttag'
+
+import {
+  JIcon,
+  JLink,
+} from 'components/base'
+
+import menuPanelStyle from './menuPanel.m.scss'
+
+export function Actions() {
+  return (
+    <nav className={menuPanelStyle.actions}>
+      <ol>
+        <li>
+          <JLink
+            href='/'
+            className={menuPanelStyle.action}
+            activeClassName={menuPanelStyle.active}
+          >
+            <JIcon name='home' size='medium' />
```

Comment:
Also no size please. Icon already contains its size, there is no need to specify it directly.